Misc/NEWS.d/next/Library/2019-02-11-18-35-06.bpo-35889.Wkcomm.rst

```diff
@@ -0,0 +1 @@
+Add ``repr()`` to the ``sqlite3.Row`` class.
```

Comment:
This can be reworded to note the change done i.e. returns a dict representation of the row.

Reply:
Done.